modules/shared/host-info.bash

```diff
@@ -1,4 +1,4 @@
-flake=$(nix flake metadata self --json | jq -r '.path' | sed -e 's|/nix/store/||' -e 's|-source||')
```

Comment:
@Mic92 this was broken by the 2.28.2 update.
```
$ nix flake metadata self
error: cannot find flake 'flake:self' in the flake registries

$ nix registry list
system flake:nixpkgs path:/nix/store/pp22j0fx88a4dnz06z0hli0kaj0jsv4n-source
system flake:self path:/nix/store/0jd6nr3xn1ib9jb886723isclvaz04cy-source
```

Reply:
@zowoq did 2.28.1 still works?

Reply:
Haven't yet tested, but I suspect this is fixed by? https://github.com/NixOS/nix/pull/13076

Reply:
Seems to fix the issue.

Reply:
Yes, that fixes it, thanks. Sorry for missing the issue that was opened a couple of days ago.